src/cli/commands/help.js

```diff
@@ -17,24 +21,47 @@ export function run(
   const getDocsInfo = (name) => 'Visit ' + chalk.bold(getDocsLink(name)) + ' for documentation about this command.';
 
   if (args.length) {
-    const helpCommand = hyphenate(args[0]);
-    if (commands[helpCommand]) {
-      commander.on('--help', () => console.log('  ' + getDocsInfo(helpCommand) + '\n'));
-    }
-  } else {
-    commander.on('--help', () => {
-      console.log('  Commands:\n');
-      for (const name of Object.keys(commands).sort(sortAlpha)) {
-        if (commands[name].useless) {
-          continue;
+    const commandName = camelCase(args.shift());
+
+    if (commandName) {
+      const command = commands[commandName];
```

Comment:
Yep! It will fix this:

```
❯ [mael-mbp?] yarn git:(0.23-stable) ❯ yarn constructor

Using globally installed version of Yarn
yarn constructor v0.23.0
error An unexpected error occurred: "command.run is not a function".
info If you think this is a bug, please open a bug report with the information provided in "/Users/mael/yarn/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/constructor for documentation about this command.
```